### DISCUSSION TITLE
How to switch between objects (e.g., plotly figure and trame `vtkRemoteView`) within one `vuetify.VContainer` (e.g., using a dropdown)?

### DISCUSSION THREAD

**Discussion by cardinalgeo** [Kitware/trame#211] [Q&A] [answered]
Hi Trame community! 

As the title suggests, I'd like to switch out the contents of a trame dashboard (e.g., using a dropdown button). For example, take the demo found [here](https://github.com/Kitware/trame/blob/master/examples/06_vtk/Applications/RemoteSelection/app.py) in the trame repo, in which selections on a vtk-rendered cylinder highlight the corresponding points in a plotly scatter plot and vice versa. 

I'd like to change this example so that, instead of two columns (`vuetify.VCol`) — the left one containing a cylinder rendered with `vtkRemoteView` and the right a scatter plot done with plotly — there'd be one container (`vuetify.VContainer`), which could be occupied by either the cylinder or the scatter plot, as determined by a dropdown. However, the necessary context/use of `with` when constructing the dashboard is throwing a wrench into this plan (i.e., I can't just define the `vtkRemoteView` object elsewhere and then switch it out). Do you have any suggestions for how I might get around this issue?

**Comment by jourdain** [comment, marked as answer]
Put them both and use `v_show=("view_mode == 'view3d'",)` so only one is up

**Comment by cardinalgeo** [comment]
Thanks @jourdain, I'll give that a try and report back!

**Reply by cardinalgeo** [reply]
This worked, thanks again for the suggestion :)
